ILSpy.ReadyToRun/ReadyToRunLanguage.cs

```diff
@@ -103,10 +103,82 @@ public override void WriteCommentLine(ITextOutput output, string comment)
 			output.WriteLine("; " + comment);
 		}
 
+		private Dictionary<VarLocType, HashSet<Tuple<DebugInfo, NativeVarInfo>>> WriteDebugInfo(ReadyToRunMethod readyToRunMethod, ITextOutput output)
```

Comment:
Can we use tuple syntax/`ValueTuple` here? Naming the tuple elements would also be nice for readability.